packages/expo-image/src/Image.types.ts

```diff
@@ -522,12 +522,15 @@ export declare class ImageRef extends SharedRef {
    */
   readonly height: number;
   /**
-   * If you load an image from a file whose name includes the `@2x` modifier, the scale is set to **2.0**.
+   * On iOS, if you load an image from a file whose name includes the `@2x` modifier, the scale is set to **2.0**.
    * All other images are assumed to have a scale factor of **1.0**.
```

Comment:
I guess a line break is not great now. It seems a bit confusing if it still refers only to iOS or all platforms. Maybe we should reword this?
```suggestion
   * On iOS, if you load an image from a file whose name includes the `@2x` modifier, the scale is set to **2.0**. All other images are assumed to have a scale factor of **1.0**.
```